### PR TITLE
feat: fix issues / styling

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- Addresses issues mentioned in https://github.com/cowprotocol/cowswap/pull/4822#issuecomment-2306978009

Screenshot 2024-08-23 at 15 50 25
Screenshot 2024-08-23 at 15 49 39

### CODE REVIEW

File: apps/cowswap-frontend/src/common/pure/OrderProgressBarV2/index.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ import { SurplusData } from 'common/hooks/useGetSurplusFiatValue'
 import { getIsCustomRecipient } from 'utils/orderUtils/getIsCustomRecipient'
 
 import * as styledEl from './styled'
-const IS_DEBUG_MODE = false
+const IS_DEBUG_MODE = true
```
Comment (alfetopito) [inline apps/cowswap-frontend/src/common/pure/OrderProgressBarV2/index.tsx:46]: Let's not forget to remove this before merging

**Review by alfetopito — APPROVED** [review]

**Review by elena-zh — APPROVED** [review]
Cannot check a transition issue as there is a debug mode is ON, but the rest 2 issues look good to me!

**Comment by fairlighteth** [comment]
updated now to remove debug mode

**Comment by elena-zh** [comment]
@fairlighteth , see how images and animations look like in iOS, real device
image
image
image

**Pull request merged**